objects.js

```diff
@@ -8520,6 +8523,48 @@ WatcherMorph.prototype.userMenu = function () {
     return menu;
 };
 
+WatcherMorph.prototype.valueExporter = function (format) {
+    var myself = this,
+        value = this.currentValue,
+        contents,
+        format = format || 'plain',
+        ide = myself.parentThatIsA(IDE_Morph);
+
+    return function () {
+        switch (format) {
+            case 'plain':
+                contents = value instanceof List ? 
+                    value.asArray().join(', ') : 
```

Comment:
Thanks @bromagosa,
I don't want to be so heavy... only a comment.
In my example, semicolons (or others) are the separators of the inner list items and newLine is the separator of the items of the main list. Then 1D lists have only newLine separators. You have seen the [example pictures](https://github.com/jmoenig/Snap--Build-Your-Own-Blocks/pull/1670#issuecomment-281836503).

Just only a comment  (discussing about this import/export feature)... not a work request :)